client/src/components/navbar-component/Dropdown.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import "./Dropdown.css";
+import DropdownItem from "./DropdownItem";
+
+function Dropdown({ isAdmin }) {
+  return (
+    <div className="dropdown-container">
```

Comment:
set z-index to some high number so that despite any other components being rendered on the page, the dropdown can be seen.